doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -736,6 +736,7 @@ Documentation
   * :ref:`build_pgm_nrf9160` section in the :ref:`ug_nrf9160` documentation by adding |VSC| and command line instructions.
   * Restructured the :ref:`programming_thingy` and :ref:`connect_nRF_cloud` sections in the :ref:`ug_thingy91_gsg` documentation.
   * The instructions and images in the :ref:`ug_thingy91_gsg` and :ref:`ug_nrf9160_gs` documentations about also accepting :term:`eUICC Identifier (EID)` when activating your iBasis SIM card from the `nRF Cloud`_ website.
+  * Screenshot in :ref:`provisioning_cloud_certificate` on the :ref:`ug_nrf9160` page to show correct application.
```

Comment:
Since this is a minor modification that the customer doesn't necessarily need to be aware of, I believe we could skip adding it to the change log.

Reply:
I was on the fence on that myself. Took the mention out.